pkgs/tools/networking/ebpf-verifier/default.nix

```
@@ -0,0 +1,64 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, substituteAll
+, pkg-config
+, cmake
+, boost
+, yaml-cpp
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "ebpf-verifier";
+  version = "2023-07-15";
+
+  src = fetchFromGitHub {
+    owner = "vbpf";
+    repo = "ebpf-verifier";
+    # no major release yet
+    rev = "de14d3aa3cd2845b621faf32b599766a66e158cf";
+    hash = "sha256-gnxB8ZLbTyIYpd61T57LPKFm1MHufeVEq/qN9pu2Vpk=";
+    fetchSubmodules = true;
+  };
+
+  patches = [
+    # prevent CMake from trying to get libraries on the Internet
+    (substituteAll {
+      src = ./remove-fetchcontent-usage.patch;
+      # Replace with catch2_3 when merged: https://github.com/NixOS/nixpkgs/pull/243485
+      catch2_src = fetchFromGitHub {
+        owner = "catchorg";
+        repo = "Catch2";
+        rev = "ac93f1943762f6fc92f0dc5bac0d720a33a27530";
+        hash = "sha256-2N+PJEfU3A2HY4AARR+PHiENTSlvzlJoObDARDJl15A=";
+      };
```

My suggestion would be to put this on all-packages.nix:

```suggestion
      catch2Src = catch2.src;
```